source/blender/makesrna/intern/rna_userdef.c

```diff
@@ -4041,6 +4048,13 @@ static void rna_def_userdef_system(BlenderRNA *brna)
 	                         "(unsafe with blend files from an untrusted source)");
 	RNA_def_property_update(prop, 0, "rna_userdef_script_autoexec_update");
 
+	prop = RNA_def_property(srna, "use_components_auto_execute", PROP_BOOLEAN, PROP_NONE);
+	RNA_def_property_boolean_negative_sdna(prop, NULL, "flag", USER_COMPONENT_AUTOEXEC_DISABLE);
+	RNA_def_property_ui_text(prop, "Auto Run UPBGE Components",
```

Comment:
You could rename "UPBGE Components" to "python components" as in the description.